portugues/simpatia/incenso/incenso.md

```diff
@@ -0,0 +1 @@
+Você desmaia com o cheiro do incenso e depois de algumas horas desacordado, você volta a si e descobre que o incenso caiu no computador, e botou fogo em tudo. Por sorte seu vizinho sente o cheiro de queimado, chama os bombeiros e te leva para o hospital. Parabéns, o bug não se resolveu, mas você continua vivo!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.